porch/api/porch/v1alpha1/types_packagerevisions.go

```diff
@@ -102,6 +103,7 @@ type Task struct {
 	Init  *PackageInitTaskSpec  `json:"init,omitempty"`
 	Clone *PackageCloneTaskSpec `json:"clone,omitempty"`
 	Patch *PackagePatchTaskSpec `json:"patch,omitempty"`
+	Edit  *PackageEditTaskSpec  `json:"edit,omitempty"`
```

Comment:
The test failure is likely because corresponding API change needs to be made to the unversioned API in the parent directory.

Reply:
Yeah, that fixed the issue.